static/css/learnocaml_common.css

```
@@ -14,6 +14,20 @@ code, pre, textarea {
   font-size: 18px;
   line-height: 18px;
 }
+
+div.anon-id a.student::before {
+  content: attr(data-anon);
+}
+
+div.token-id a.student::before {
+  content: attr(data-token);
+}
+
+div.nickname-id a.student::before {
+  content: attr(data-nickname);
+}
```

Why not using some `div`s ?

```suggestion
div.anon-id div.student::before {
  content: attr(data-anon);
}

div.token-id div.student::before {
  content: attr(data-token);
}

div.nickname-id div.student::before {
  content: attr(data-nickname);
}
```

But maybe your choice was necessary, anyway I'll be happy to talk about this next time :)